controller/controller_manager.go

```diff
@@ -148,18 +148,32 @@ func GetInstanceManagerCPURequirement(ds *datastore.DataStore, imName string) (*
 	}
 
 	cpuRequest := lhNode.Spec.InstanceManagerCPURequest
-	guaranteedCPUSettingName := types.SettingNameGuaranteedInstanceManagerCPU
 	if cpuRequest == 0 {
-		guaranteedCPUSetting, err := ds.GetSettingWithAutoFillingRO(guaranteedCPUSettingName)
-		if err != nil {
-			return nil, err
+		switch im.Spec.BackendStoreDriver {
+		case longhorn.BackendStoreDriverTypeV1:
+			guaranteedCPUSetting, err := ds.GetSettingWithAutoFillingRO(types.SettingNameGuaranteedInstanceManagerCPU)
```

Comment:
todo: Need to update the doc to mention this is for v1 only.

Reply:
Doc is added. Thank you.
https://github.com/longhorn/website/pull/825